HyphaApp/hypha
# Report reminders

Fixes #3989

## Test Steps

Set up a fake SMTP server or use file-based email.

- Configure reports for projects in Wagtail Admin
- See that report reminders go out as configured
